listenbrainz_spark/recommendations/create_dataframes.py

```diff
@@ -49,10 +49,9 @@ def training_data_window():
         Note: Under the assumption that config.TRAIN_MODEL_WINDOW will always indicate months.
     """
     training_df = None
-    # go to first of the current month and then decrement months
-    d1 = stats.replace_days(1)
-    d2 = stats.adjust_months(d1, -config.TRAIN_MODEL_WINDOW)
-    training_df = utils.get_listens(d2, d1)
+    current_date = datetime.utcnow()
+    adjusted_date = stats.adjust_months(current_date, -config.TRAIN_MODEL_WINDOW)
+    training_df = utils.get_listens(current_date, adjusted_date)
```

Comment:
should I change current_date and adjusted_date to begin_date and end_date here also?